.claude/commands/start.md

```
@@ -1,107 +1,301 @@
+  "ticket_description": "Description from issue tracker",
+  "ticket_status": "In Progress"
+}
+```
+
+## Step 4: Create Branch
+
+### Generate Branch Name
+
+Apply the branch pattern from config (default: `{type}/{ticket}-{description}`):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)